controller/src/main.cpp

```diff
@@ -60,6 +61,31 @@ static ControllerStatus controller_status = ControllerStatus_init_zero;
 // Last-received status from the GUI.
 static GuiStatus gui_status = GuiStatus_init_zero;
 
+// desired actuators state
+static ActuatorsState actuators_state;
+
+class Controller {
```

Comment:
I am imagining that the reason you added this new class that only has one member function and no member variables is because you're envisioning that it will have a bunch more stuff at some point in the near future?

I wonder if we could either (a) embrace YAGNI and add the class only when we need it, or (b) explain our vision in a comment.  My preference would be either (a) or both, i.e. don't add the class until it makes sense with the code as-is, but if you want to write down in words your vision for what should happen here, that could help the person who goes to make this transformation.

Reply:
Will add comments to this: this is what we need to interface with Modellica.

Reply:
OK.  Modelica requires that it's a class; it can't be a function?

Reply:
We can integrate anything we want with Modelica, its API is just C functions for constructing, stepping and destructing. But I think it'd be good to have this be a class with no static state anywhere underneath it, + a static instance of the class in controller main (and a non-static one when run from Modelica). Not necessary to get it all the way to that form in this PR though.

As a nice benefit, it becomes unit-testable this way.